tasks/check.yml

```diff
@@ -28,3 +28,10 @@
 #     path: /usr/share/upstart
 #   register: upstart_check
 
+
+- name: Set service_opts if undefined
+  when: service_opts is undefined and amq_broker_instance_dir is defined
```

Comment:
This is a common ansible role which maybe wants to be used by anyone. I can't approve it during your product dependency, sorry... We had only qdrouterd used in README.md examples.